lib/client-assets.php

```diff
@@ -693,3 +703,15 @@ function gutenberg_extend_settings_custom_units( $settings ) {
 	register_block_pattern( 'core/its-time', gutenberg_load_block_pattern( 'its-time' ) );
 	register_block_pattern( 'core/testimonials', gutenberg_load_block_pattern( 'testimonials' ) );
 }
+
+/*
+ * Register default pattern categories if not registered in Core already.
+ */
+if ( class_exists( 'WP_Block_Pattern_Categories_Registry' ) ) {
+	register_block_pattern_category( 'cover', [ 'label' => __( 'Cover' ) ] );
```

Comment:
I think we should add a "text" category and have the text column present in both